src/makeRobotAccountant.js

```diff
@@ -6,7 +6,24 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+
+  const add = (number) => {
+    if (callCount >= 3) {
+      callCount++;
+
+      if (callCount % 2 === 0) {
+        return () => 'Bzzz... Error!';
+      }
+
+      return nextNumber => number + nextNumber;
+    }
+    callCount++;
+
+    return nextNumber => number + nextNumber;
```

Comment:
checklist 5. [CODE STYLE] - DRY. When you do similar logic, try to combine conditions

the same operations as in `if (callCount >= 3) {}`. It possible to use only one if

Reply:
Thanks, good point)